net-p2p/bitcoin-core/bitcoin-core-25.1-r1.ebuild

```diff
@@ -144,23 +181,41 @@ src_configure() {
 		--enable-util-tx
 		--${wallet}-util-wallet
 		--disable-util-util
+		--without-seccomp
 		$(use_with libs)
 		$(use_with daemon)
-		$(use_with gui)
+		$(use_with gui gui qt5)
 		$(use_with dbus qtdbus)
 		$(use_with system-leveldb)
 		$(use_with system-libsecp256k1)
 	)
 	econf "${myeconfargs[@]}"
 }
 
+src_test() {
+	emake check
+
+	if use daemon ; then
+		local jobs=$(get_makeopts_jobs unset)
+		[[ "${jobs}" == unset ]] && unset jobs
+		set -- "${PYTHON}" test/functional/test_runner.py --ansi --extended ${jobs+--jobs="${jobs}"} --timeout=0
+		echo "${@}" >&2
+		"${@}" || die 'test/functional/test_runner.py failed'
+	fi
+}
+
 src_install() {
 	use external-signer && DOCS+=( doc/external-signer.md )
 	use berkdb || use sqlite && DOCS+=( doc/managing-wallets.md )
 	use libs && DOCS+=( doc/shared-libraries.md )
 	use systemtap && DOCS+=( doc/tracing.md )
 	use zeromq && DOCS+=( doc/zmq.md )
 
+	if use daemon ; then
+		DOCS+=( share/rpcauth/rpcauth.py )
+		docompress -x "/usr/share/doc/${PF}/rpcauth.py"
```

Comment:
maybe add a bug reference in a comment given there's some useful discussion in the bug (someone may well come along in a while and ask why we're installing it)